### ISSUE TITLE
Open-Ocr-Client returns the incorrect text

### ISSUE THREAD

**Issue by williamczuba** [tleyden/open-ocr-client#2]
When I run the open-ocr-client routing to <HOSTING>/ocr-file-upload, and pass in a local image, it outputs: 

```
22:55:24.355770 results: <h1>OpenOCR is running!<h1> Need <a href="http://www.openocr.net">docs</a>?
22:55:24.355798 err: <nil>
```

When I use curl, and pass in a url of the same image, I get the correct OCR of:
     You can create local variables for the pipelines within the template by
     preﬁxing the variable name with a “$" sign. Variable names have to be
     composed of alphanumeric characters and the underscore. In the example
     below I have used a few variations that work for variable names.

What could be causing this, and how can I fix it?

**Comment by williamczuba** [comment]
I actually managed to fix it.  It had to do with how the URL for the request was constructed.  I went into the main.go for the cli-main directory, and changed the openOCRClient to:

```
 openOcrClient := ocrclient.NewHttpClient(*ocrUrl)
```

This was necessary after hosting the project on Google Cloud, let me know if anyone runs into a similar issue and needs a hand.  I'm happy to help if I can find the time.

**Comment by tleyden** [comment]
@wigywizzle what was the code before you changed to

```
 openOcrClient := ocrclient.NewHttpClient(*ocrUrl)
```

?

**Comment by williamczuba** [comment]
I apologize, that's not where I made my code modification.  I actually manually entered in the correct url routing for using the OCR with an image (ie: /ocr-file-upload), and changed 

```
apiUrl := c.OcrApiFileUploadEndpointUrl()
```

to 

```
apiUrl := c.ApiEndpointUrl
```

in ocr_http_client.go's DecodeImageReader function, since the endpointUrl function was actually adding the routing to the end of the URL:PORT instead of to the end of the URL.  This was setting the URL for the post to be URL:PORT/ocr-file-upload instead of URL/ocr-file-upload:PORT, which cause automatic rerouting to the main page in the open-ocr that returns the weird output that I was getting.
